### PR TITLE
Adds Sierra Power armor and gives it to the NCR Captain

### PR DESCRIPTION
**Pull request by JohnOxford**
## Description
Adds Sierra Power armor and gives it to the NCR Captain

## Motivation and Context
Puts the captain on tier with the rest of the faction leaders and distinguishes him from his lieutenant, maybe one day i'll sprite a helmet

## How Has This Been Tested?
yes
## Screenshots (if appropriate):
39e09c1651516b68a2ef58e436622132

### CODE REVIEW

**Comment by JohnOxford** [comment]
you have to consider what's more important though, the leader of a faction being able to stack up against other faction leaders, being distinguishable, and really how neato sierra pa is. 

you'd also spend less time grinding to become a knight with a identical armor set to this (15 hours) compared to the captains 24. 

then again we could just ban the generically awful people who powergame for the roles.

i'll sit on it and if no one else inputs i'll just remove the captain spawning with it

**Comment by UndeadNappist** [comment]
> you have to consider what's more important though, the leader of a faction being able to stack up against other faction leaders, being distinguishable, and really how neato sierra pa is.

I love the HK CAWS too, it looks good and someone did a whole bunch of functional back sprites for it and a bunch of other stuff. For many, many other guns as well (see #471)
Still, the captain IS recognizable through his unique costume and hat. The same for the vet rangers. Imagery is important.

> then again we could just ban the generically awful people who powergame for the roles.

I would love for that to happen. At the time, it wasn't, and there was pressure against that, and simply making it impossible and putting pressure on players to behave mechanically is a lot stronger than just saying no in rules they may or may not read.

**Comment by RenGusta** [comment]
Just a reminder that the Sierra was one of the few intact autos the NCR would have, and it was in the possession of the Commanding Officer of the Long 15. The NCRs most vital supply and trade route into the Mojave.

**Comment by JohnOxford** [comment]
on the same hand, since there's only one NCR battalion on the entire map, and there can only be one captain in charge of it, and generally a commanding officer is a captain, it's not -that- far of a stretch to say that he would be in possession of it

if not, then by god we have to readd the colonel role so admins can at minimum make usage of this in events

**Comment by JohnOxford** [comment]
gonna fix this and make it so the colonel spawns with it assuming that the job still exists, since royez was a colonel not a captain.

pls merge after so it doesnt break again

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -55,18 +69,19 @@ Captain
...
-	glasses = 		/obj/item/clothing/glasses/sunglasses/big
-	suit_store = 	/obj/item/gun/ballistic/automatic/pistol/m1911
+	jobtype 	= /datum/job/ncr/f13captain
+	uniform		= /obj/item/clothing/under/f13/ncr/officer
+	accessory 	= /obj/item/clothing/accessory/ncr/CPT
+	suit 		= /obj/item/clothing/suit/armor/f13/ncrarmor/mantle
+	head 		= /obj/item/clothing/head/beret/ncr
+	gloves 		= /obj/item/clothing/gloves/combat
+	glasses 	= /obj/item/clothing/glasses/sunglasses/big
+	suit_store  = /obj/item/gun/ballistic/automatic/marksman
```
Comment (UndeadNappist) [inline code/modules/jobs/job_types/ncr.dm]: Do not give the captain the /automatic/marksman. It's a fully-automatic sniper rifle.

**Pull request merged**